framework/src/modules/dpos/dpos_module.ts

```diff
@@ -175,6 +167,23 @@ export class DPoSModule extends BaseModule {
 		}
 
 		await setRegisteredDelegates(context.stateStore, { registeredDelegates: delegateUsernames });
+
+		const initialRound = this.rounds.calcRound(context.genesisBlock.header.height) + 1;
```

Comment:
Suggest to use some name other than `initialRound`, its got mixed up with the `asset.initRounds` property.